### PR TITLE
Merge sleepprogger prs

### PR DESCRIPTION
**Pull request by brianretford**

### CODE REVIEW

**Comment by SleepProgger** [comment]
Thank you for merging. I had fun figuring out Tile.

You seem to have missed https://github.com/plaidml/plaidml/pull/291/commits/c06cbfd4aa76fe5850102cd91a4ea437cff34224 tho, which added `name` and `seed` support for `random_normal_variable`

File: plaidml/tf_test.py [hunk 1]
```diff
@@ -116,14 +126,28 @@ class TestTF(unittest.TestCase):
...
 
     @opTest([
-        [np.random.random((5, 60, 10, 3)), [1,3,4,1], [1,3,3,1], [1,1,1,1], "SAME"],
-        [np.random.random((5, 60, 10, 3)), [1,3,4,1], [1,3,3,1], [1,1,1,1], "VALID"],
-        [np.random.random((5, 60, 10, 3)), [1,4,3,1], [1,1,1,1], [1,1,1,1], "SAME"],
-        [np.random.random((5, 60, 10, 3)), [1,4,3,1], [1,1,1,1], [1,6,5,1], "SAME"],
-    ])
-    def testExtractImagePatches(self, b, images, ksizes, strides, rates=(1,1,1,1), padding="VALID"):
-        func = tensorflow.extract_image_patches if b == tf else plaidml.op.extract_image_patches
-        return [func(images, ksizes, strides, rates, padding)]
```
Comment (SleepProgger) [inline plaidml/tf_test.py]: This was probably accidentally removed ?

Comment (brianretford) [reply plaidml/tf_test.py]: Yup, restored in 0.6.1, also put tf_test into bazel so we run it in CI

**Comment by brianretford** [comment]
Merged c06cbfd into master

**Pull request merged**